src/tools/admin/conseillers/desinscription-recrutes.js

```diff
@@ -202,6 +202,16 @@ execute(__filename, async ({ db, logger, exit, emails, Sentry, gandi, mattermost
                 }
               );
 
+              //Suppression des permanences ne comportant que le conseillerCoop en rupture
+              const permanences = await db.collection('permanences').find({ 'conseillers': { '$in': [conseillerCoop._id] } });
+              for (let permanence of permanences) {
+                if (permanence?.conseillers?.length === 1) {
+                  await db.collection('permanences').deleteOne({
+                    '_id': permanence._id
+                  });
+                }
+              }
```

Comment:
Normalement ca doit passer en 1 requete : 
await db.collection('permanences').deleteOne(
{ 'conseillers': { '$in': [conseillerCoop._id] } },
{ 'conseillers': { '$size': 1 } }, (à vérifier ici si pas besoin d'un $and)
);

ou même :
await db.collection('permanences').deleteOne(
{ 'conseillers': { '$eq': [conseillerCoop._id] } }
);

Reply:
Effectivement !